opencti-platform/opencti-front/src/utils/filters/filtersUtils.tsx

```diff
@@ -129,6 +129,7 @@ export const stixFilters = [
   'takedown_types',
   'note_types',
   'incident_type',
+  'inPir',
```

Comment:
do we want this filter in stream and triggers ? or only playbooks? If only for playbooks, add 'inPir' only in PlaybookAddComponent in case the selected component is the new pirComponent